docs/test_runners.md

```diff
@@ -0,0 +1,48 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+1. Freshness (Valoraremos esto con `snyk`).
+2. Como de completo es el `framework` para satisfacer las necesidades del proyecto.
```

Comment:
Si no dices cuales son las necesidades, esto no deja de ser un criterio arbitrario. Por ejemplo, si estuvieras trabajando en TypeScript sería razonable que pudiera trabajar con ese lenguaje; o si usaras web sería razonable que pudiera trabajar con el DOM. Sin especificar cuales son esas necesidades, no deja de ser un criterio arbitrario que vas a usar para que "salga" el que tenías pensado desde el principio.

Reply:
La verdad que no sé si estoy sugestionado o no, porque lo analizo todo lo objetivamente, me meto en muchos foros para comparar entre las alternativas, me leo los manuales y características y me sigue pareciendo mejor alternativa `Jest`. Es más simple de poner a configurar y es la opción que tiene mejor mantenimiento excepto `ava`, la cual es demasiado minimalista, además de no tener estilo BDD, lo que me parece una característica negativa.

Reply:
En informática nunca hay una "mejor" alternativa. `jest` no es mejor que `mocha` o que usar `ava` y `node:test`, `node:assert` y las aserciones de `ava`. El error empieza por ahí, por pensar que tenéis que acertar. Se trata como ya os dije que sepáis establecer criterios y tomar una decisión con base técnica de forma adecuada, contemplando todas las alternativas posibles. Trabajar desde "el mejor" hacia atrás, a buscar criterios y listas que lo incluyan es donde está el problema.